src/autoscan.cc

```diff
@@ -174,30 +168,28 @@ int AutoscanList::removeByObjectID(int objectID)
 {
```

Comment:
Whole function `removeObjectById` is not used. nuke it.

Reply:
Nuked.